services/frontend/src/components/AnalysisResultTable.vue

```diff
@@ -11,15 +11,15 @@
       </tr>  
       <tr 
         v-bind:key="record.analysisId" 
-        v-for="record in analyses"
+        v-for="record in analysis"
       >
         <td>{{record.analysisId}}</td>
         <td>{{ `${record.patientId} ${record.patientIdType}` }}</td>
         <td>{{record.patientName}}</td>
         <td>{{record.patientSurname}}</td>
         <td>{{record.date}}</td>
         <td>
-          <router-link to="/analysis-view" v-bind:href="record.analysisHyperlink">
+          <router-link to='/doctor/analysis-view' >
```

Comment:
I would like to do deeper consideration one page design vs multi page design, before this can be merged